resources/configurationFiles/analysisIndelCalling.xml

```diff
@@ -53,6 +53,10 @@
         <cvalue name="LOCAL_CONTROL_COMMON_SNV" value="${hg19DatabasesDirectory}/LocalControls/pcawg_AF.All.CommonArtifacts.vcf.gz" type="path"/>
         <cvalue name="LOCAL_CONTROL_COMMON_SNV_2" value="${hg19DatabasesDirectory}/LocalControls/LocalControlConsolidated_AF.INFO.ACgt10.vcf.gz" type="path"/>
         <cvalue name="EXOME_CAPTURE_KIT_BEDFILE" value='${hg19DatabasesDirectory}/../targetRegions/Agilent5withUTRs_plain.bed.gz' type="path"/>
+        <!-- New gnomAD files and local control, needs to be moved to NGS share -->
+        <cvalue name="GNOMAD_V2_1_GENOME_SNV_INDEL" value="/icgc/dkfzlsdf/analysis/UniExome/GermlineAnnotation/annotationInfo/gnomAD/v2.1/genomes/gnomad.genomes.r2.1.sites.SNV-INDEL.vcf.gz" type="path" />
```

Comment:
Should this not also given as relative path? Should the files not get installed in the ngs_share?

Reply:
Also: I think at some point we decided to not have internal paths in workflow repos (even not for public repos like this one -- the public part is also problematic). Maybe you could introduce a variable similar to `hg19DatabaseDirectory` just upstream the internal DB download structure. E.g. Downloaded was the stuff below `v2.1`, thus everything left of it would be a variable pointing to the gnomad database directory base-directory. This way you have documented (1) how to configure the DB, (2) which files from the DB to use. Then you add an example link to the gnomAD FTP server to the `description` attribute for downloading and everything is fully documented.

Reply:
Files are moved to NGS share and the config file is now updated.